### PR TITLE
Added 'status' parameter to formatFormInput function

### PR DESCRIPTION
**Pull request by matt-paul**
...to allow different status in post and update member functions.  This will stop multiple confirmation emails being sent.

### Motivation

- [Related story](http://...)

TODO: write a motivation.

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by chris-brookes** [comment]
More changes still to be done.

**Comment by chris-brookes** [comment]
@callerc1 - Martin has done a number of changes to this PR since the last review. Can you give it another quick review please.

File: services/mailchimp/config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const groups = {
-  techPageSignup: '17dfd6ce16',
+  techPageSignup: process.env.ENVIRONMENT_NAME === 'live' ? '17dfd6ce16' : 'd2e5a52843',
```
Comment (callerc1) [inline services/mailchimp/config.js]: Not sure if this is the correct env var we should be using. Like is this actually an env var in the lamda process? May be worth double checking how this works. I have some ideas around some alternative approaches for the mailchimp group mapping. Let have a chat when you get around to it.

File: services/mailchimp/config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const groups = {
-  techPageSignup: '17dfd6ce16',
+  techPageSignup: process.env.MAILCHIMP_TECH_GROUP_ID,
```
Comment (callerc1) [inline services/mailchimp/config.js]: way better solution imo 👍

**Pull request merged**